### PR TITLE
Fixes for domu creation and use it shared disk or network

### PR DESCRIPTION
**Pull request by xakep-amatop**

### CODE REVIEW

File: xen-shell-cmd/src/xen_cmds.c [hunk 1]
```diff
@@ -48,7 +49,12 @@ static int domu_create(const struct shell *shell, int argc, char **argv)
 	 * TODO: this should be changed in app code.
```
Comment (xakep-amatop) [inline xen-shell-cmd/src/xen_cmds.c:49]: note: this TODO will be resolved in the next PR

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Review by arminn — APPROVED** [review]
Reviewed-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

**Review by firscity — APPROVED** [review]
`Reviewed-by: Dmytro Firsov <dmytro_firsov@epam.com>`

**Comment by xakep-amatop** [comment]
added acks to commits

**Pull request merged**